harvester/sharekit/tests/test_get_harvest_seeds.py

```diff
@@ -0,0 +1,158 @@
+from datetime import datetime
+
+from django.test import TestCase
+from django.utils.timezone import make_aware
+
+from harvester.utils.extraction import get_harvest_seeds
+from sharekit.tests.factories import SharekitMetadataHarvestFactory
+from sharekit.extraction import SHAREKIT_EXTRACTION_OBJECTIVE
+
+
+class TestGetHarvestSeedsSharekit(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        SharekitMetadataHarvestFactory.create_common_sharekit_responses(include_delta=True)
+
+    def extract_seed_types(self, seeds):
+        normal = next(
+            (seed for seed in seeds
+             if seed["state"] != "deleted" and "maken.wikiwijs.nl" not in seed["url"])
```

Comment:
Waarom krijgt maken.wikiwijs.nl hier een bijzondere behandeling?

Reply:
Omdat het een IMSCC package is en die krijgen links en rechts wat extra's. Tegenwoordig niet meer, dus ik neem aan dat we het kunnen deleten. Kan ik even naar kijken.